app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const cors = require('cors')
+const app     = express();
+
+require('dotenv').config()
+const host = process.env.NODE_ENV === "production" ? process.env.HOST : 'http://subdomain.jscms.test:6000/';
+const port = process.env.PORT || 6000;
+
+const whitelisted = ['http://localhost:6000/', 'http://jscms.test/'];
+const corsOptionsDelegate = function (req, callback) {
```

Comment:
Awesome. Can we use es6 standards here? 
e.g: `const corsOptionsDelegate = (req, callback) => {...}`